crates/contracts/src/multisend.rs

```diff
@@ -0,0 +1,53 @@
+//! Module for encoding transactions for the Safe periphery `MultiSend` and
+//! `MultiSendCallOnly` contracts.
+//!
+//! Details about the encoding can be found:
+//! - <https://github.com/safe-global/safe-contracts/blob/v1.3.0/contracts/libraries/MultiSend.sol#L17-L23>
+//! - <https://github.com/safe-global/safe-contracts/blob/v1.3.0/contracts/libraries/MultiSendCallOnly.sol#L10-L16>
+
+use ethcontract::{Bytes, H160, U256};
+
+#[derive(Clone, Debug, Default)]
+pub struct Transaction {
+    pub op: Operation,
+    pub to: H160,
+    pub value: U256,
+    pub data: Vec<u8>,
+}
+
+#[derive(Clone, Copy, Debug, Default)]
+#[repr(u8)]
+pub enum Operation {
+    #[default]
+    Call = 0,
+    DelegateCall = 1,
+}
+
+pub fn encode(transactions: &[Transaction]) -> Bytes<Vec<u8>> {
```

Comment:
Would it make sense to link the trampoline contract code here so readers can verify that this encoding is correct?

Reply:
I added links in the module doc-comment: https://github.com/cowprotocol/services/blob/f24fa615d7f499e48b5f4995499da2045706c704/crates/contracts/src/multisend.rs#L4-L6

I can move them here so its more obvious to find.

Reply:
21d71f13528f0934c11ba865db39342e781e95ed